Moq.Dapper.Test/Moq.Dapper.Test.csproj

```diff
@@ -1,82 +1,34 @@
+    <Version>1.0.0.7</Version>
+    <PackageId>Moq.Dapper</PackageId>
+    <Description>Moq extensions for Dapper methods.</Description>
+    <Copyright>Copyright 2017</Copyright>
+    <FileVersion>1.0.0.0</FileVersion>
+    <Company>Stefano d'Antonio</Company>
+    <Authors>Stefano d'Antonio</Authors>
+    <Product>Moq.Dapper</Product>
+    <PackageLicenseUrl>https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html</PackageLicenseUrl>
+    <PackageProjectUrl>https://github.com/UnoSD/Moq.Dapper</PackageProjectUrl>
+    <RepositoryUrl></RepositoryUrl>
+    <PackageTags>Moq Dapper Test</PackageTags>
+    <PackageReleaseNotes>First release.</PackageReleaseNotes>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <AssemblyVersion>1.0.0.7</AssemblyVersion>
+    <IsPackable>false</IsPackable>
   </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <DebugSymbols>true</DebugSymbols>
-    <DebugType>full</DebugType>
-    <Optimize>false</Optimize>
-    <OutputPath>bin\Debug\</OutputPath>
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <DebugType>pdbonly</DebugType>
-    <Optimize>true</Optimize>
-    <OutputPath>bin\Release\</OutputPath>
-    <DefineConstants>TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-  </PropertyGroup>
-  <ItemGroup>
-    <Reference Include="Castle.Core, Version=4.0.0.0, Culture=neutral, PublicKeyToken=407dd0808d44fbdc, processorArchitecture=MSIL">
-      <HintPath>..\packages\Castle.Core.4.0.0\lib\net45\Castle.Core.dll</HintPath>
-    </Reference>
-    <Reference Include="Dapper, Version=1.50.2.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\Dapper.1.50.2\lib\net451\Dapper.dll</HintPath>
-    </Reference>
-    <Reference Include="Moq, Version=4.7.1.0, Culture=neutral, PublicKeyToken=69f491c39445e920, processorArchitecture=MSIL">
-      <HintPath>..\packages\Moq.4.7.1\lib\net45\Moq.dll</HintPath>
-    </Reference>
-    <Reference Include="nunit.framework, Version=3.6.1.0, Culture=neutral, PublicKeyToken=2638cd05610744eb, processorArchitecture=MSIL">
-      <HintPath>..\packages\NUnit.3.6.1\lib\net45\nunit.framework.dll</HintPath>
-    </Reference>
-    <Reference Include="System" />
-    <Reference Include="System.Data" />
-    <Reference Include="System.Data.Linq" />
-  </ItemGroup>
   <ItemGroup>
-    <Compile Include="DapperExecuteAsyncTest.cs" />
-    <Compile Include="DapperExecuteTest.cs" />
-    <Compile Include="DapperQueryAsyncTest.cs" />
-    <Compile Include="DapperQueryTest.cs" />
-    <Compile Include="DapperTest.cs" />
-    <Compile Include="DataContextTest.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
+    <PackageReference Include="Dapper" Version="1.50.5" />
```

Comment:
Can you please consolidate the package versions? Other projects run on 1.50.2. We could update all to the latest (Also for other packages).